README.md

```diff
@@ -18,9 +18,18 @@ npm install eslint-teamcity --save-dev
 ```
 
 ## Usage
+* as onrun eslint formatter:
 ```sh
   eslint --format './node_modules/eslint-teamcity/index.js' myfiletolint.js
 ```
+* as post runnig formatter, work with standart eslint json report
```

Comment:
it`s not helpfull because if you use reporter in this case y know how it works)
but i add it, ok)